versions/1.0.0.md

```diff
@@ -111,10 +111,11 @@ This object represents one or more changes to be applied to the target document
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="actionTarget"></a>target | `string` | **REQUIRED** A JSONPath query expression referencing the target objects in the target document.
-<a name="actionDescription"></a>description | `string` | A description of the action. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
-<a name="actionUpdate"></a>update | Any | An object with the properties and values to be merged with the object(s) referenced by the `target`. This property has no impact if `remove` property is `true`.
```

Comment:
Is it worth keeping update aroud for a minute as deprecated or is this early and experimental enough that whatever?

Reply:
This is always a hard decision to make.  At this point in time, I'd vote to just kill it.  We won't always have this luxury, but we should make use of it while we can.

Reply:
We'll always have the commit history, but this does make me wonder if we should propose a 0.1.0 version and then start following a more formal release process. We're already seeing usage ....

Reply:
.... Alternatively, how about moving forward from our current situation by making this a 1.1.0 and still a draft?

Reply:
Question retracted, you folks came up with a god solution on the call and I forget what it was, but I'm not standing in the way of anything 🫡

Reply:
This proposal will definitely be 2.0 in recognition of how breaking it is.